packages/orchestrator-ui-components/src/components/WfoBadges/WfoProcessTargetBadge/WfoProcessTargetBadge.tsx

```diff
@@ -4,11 +4,10 @@ import { useOrchestratorTheme } from '../../../hooks';
 import { WorkflowTarget } from '../../../types';
 import { WfoBadge } from '../WfoBadge';
 
-export type WfoWorkflowTargetBadgeProps = {
+export type WfoProcessTargetProps = {
```

Comment:
I had made the assumption this badge would only be used on Workflow related pages since the target for a task is always System. If are using it in other places aswell then this rename is ok but if not I think the name made sense

Reply:
I think we still show this column in the task page, so I thought make this one generic.